<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.669, y2:0.897727, stop:0 rgba(40, 66, 72, 255), stop:1 rgba(79, 123, 127, 255));}</string>
   </property>
   <widget class="QPushButton" name="btn_erstellen">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 700 18pt &quot;MS Shell Dig 2&quot;;
background-color: white;
color:black;

</string>
    </property>
    <property name="text">
     <string>Account erstellen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>261</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dig 2&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Registrieren</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>521</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dig 2&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>Registriere dich mit einem neuen Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nutzername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_passwort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nutzername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dig 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Nutzername</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dig 2&quot;;
color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Passwort</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_regex">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dig 2&quot;; color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_confirmpasswort">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dig 2&quot;;
color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Nochmal Passwort</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dig 2&quot;;
color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
